### DISCUSSION TITLE
Noob help

### DISCUSSION THREAD

**Discussion by maureenfbrennan** [sec-edgar/sec-edgar#223] [Q&A]
Hi there,

I am a Data Science student and would like to get SEC text data for a project.  This package seems amazing, but I am having trouble getting started.  I use Jupyter Notebooks on Anaconda.  I installed the secedgar package using the Anaconda Command Prompt, using: pip install secedgar.

I keep getting the error: ModuleNotFoundError: No module named 'secedgar.core'.  I have uninstalled and re-installed anaconda.  It really seems like this is a user error, but I can't figure it out.  Could anyone point me in the right direct, please?

![image](https://user-images.githubusercontent.com/87739181/126413808-0ca043f5-9797-41a8-8308-2fe236e95358.png)

Many thanks,
Maureen

**Comment by jackmoody11** [comment]
It looks like you have probably installed the latest release that isn't in alpha. We are currently working on a new version which restructures the package. To download that new version, you will want to install the alpha version of v0.4.0:

```
pip install secedgar==0.4.0-alpha
```

Let me know if this helps or if you run into any other issues.

**Reply by maureenfbrennan** [reply]
Hi Jack,

Many, many thanks for your reply.  I really appreciate it.

I followed your prompt, and it did resolve the missing secedgar.core module error.  However, I am still having trouble with the initial example.  When I try to run the example from ReadMe, I get a "no filings" error:
![image](https://user-images.githubusercontent.com/87739181/126643998-92c84a09-b6df-48e9-8c42-34a1d95f8778.png)

Could this because my setup is incorrect?

Many thanks and best regards,
Maureen

**Reply by jackmoody11** [reply]
You need to add the user_agent keyword argument to `CompanyFilings` like this:


```python

CompanyFilings(..., user_agent="Your Name (Your Email)")
```

I am in the process of making this clearer in the documentation, as the SEC now seems to require it.
